README.md

```diff
@@ -1,2 +1,18 @@
 # fps-gain-trainer
 Front-end for fps-gain-trainer
+
+## Setup
+### Prerequisites
+- Install `node >= 8.10` and have `npm >= 5.6`.
+- Next you're going to need to install React JS, `cd` into the directory where you want the project to live then run the following commands.
```

Comment:
This is actually an incorrect setup process. As long as you have your prerequisites and the package.json in this folder works, you should be able to just simply type in "npm install".

Reply:
Added an alternative way of installing it